paasta_tools/contrib/paasta_update_soa_memcpu.py

```diff
@@ -266,6 +266,7 @@ def main(argv=None):
             service_param=_get_dashboard_qs_param('paasta_service', serv['service']),
             instance_param=_get_dashboard_qs_param('paasta_instance', serv['instance']),
         )
+        summary = f"Rightsizing {serv['service']}.{serv['instance']} in {serv['cluster']} to make it not have {provisioned_state}"
```

Comment:
provisioned_state is just `over` or `under` so i think you will still need to jam `provisioned cpu` on at the end of this string